src/components/modal/modal.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, Event, EventEmitter, h, Host } from "@stencil/core"
```

Comment:
It would be my preference to not publish this as a generic component for modals, I feel like generic "modals" are an anti-pattern that ends up leading to bad situations... its not exactly rocket science to use CSS to overlay and position some HTML

Thoughts? I think maybe moving it back to just a specific `file-upload-modal` or event `file-upload-dialog` would be a better fit and less likely to end up being hard to work with later

Reply:
I thought it will be usefull to have a modal component. But i can set it back to be a specific to upload. Now, it should probably be part of the upload component and not a component itself that cannot be used by itself.